macros/plugins/databricks/create_external_table.sql

```diff
@@ -0,0 +1,41 @@
+{% macro databricks__create_external_table(source_node) %}
+
+    {%- set columns = source_node.columns.values() -%}
+    {%- set external = source_node.external -%}
+    {%- set partitions = external.partitions -%}
+    {%- set options = external.options -%}
+
```

Comment:
Love that you've created this PR. Been thinking about doing it myself for a while, but haven't gotten further than overriding the  `databricks__create_external_table` and `databricks__get_external_build_plan` macros locally.

One idea you could take into account is adding the support for liquid clustering:
![image](https://github.com/dbt-labs/dbt-external-tables/assets/19465110/23127440-dce0-4b6a-a2dc-23fbc67e437c)